### PR TITLE
Bri12415/q futurize display info

### PR DESCRIPTION
**Pull request by bri12415**
# Description

## Type of change

- [ ] Bug fix
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [x] Other enhancement

**Platforms tested on**:

- [ ] Windows
- [ ] Android
- [ ] Linux
- [x] macOS
- [ ] iOS

## Checklist

- [ ] Runs and compiles on all active platforms as a standalone sample
- [x] Runs and compiles in the sample viewer(s)
- [x] Branch is up to date with the latest main/v.next
- [x] All merge conflicts have been resolved
- [x] Self-review of changes
- [ ] There are no warnings related to changes <-- QFuture warnings
- [x] No unrelated changes have been made to any other code or project files
- [x] Code is commented with correct formatting (CTRL+i)
- [x] All variable and method names are camel case
- [x] There is no leftover commented code
- [x] Screenshots are correct size and display in description tab (500px by 500px, platform agnostic)
- [ ] If adding a new sample, it is added to the sample viewer
- [ ] Cherry-picked to Main branch (if applicable)

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/CreateSymbolStylesFromWebStyles/CreateSymbolStylesFromWebStyles.cpp [hunk 1]
```diff
@@ -64,31 +63,31 @@ CreateSymbolStylesFromWebStyles::CreateSymbolStylesFromWebStyles(QObject* parent
...
   QMap<QString,QStringList> categoriesMap = createCategoriesMap();
-  SymbolStyle* symbolStyle = new SymbolStyle("Esri2DPointSymbolsStyle", {}, this);
   SymbolStyleSearchParameters searchParams;
+  searchParams.setKeys(categoriesMap.keys());
   searchParams.setKeysStrictlyMatch(true);
 
-  connect(symbolStyle, &SymbolStyle::searchSymbolsCompleted, this, [this, symbolStyle, categoriesMap](const QUuid& /* taskId */, SymbolStyleSearchResultListModel* searchResults)
+  SymbolStyle* symbolStyle = new SymbolStyle("Esri2DPointSymbolsStyle", {}, this);
+  symbolStyle->searchSymbolsAsync(searchParams).then(this,
+  [this, symbolStyle, categoriesMap](SymbolStyleSearchResultListModel* searchResults)
```
Comment (bri12415) [inline ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/CreateSymbolStylesFromWebStyles/CreateSymbolStylesFromWebStyles.cpp:78]: I dont like the double pass through of categoriesMap, but couldnt find a quick way to refactor other than pulling categories map into a class field which I didnt think made sense. Any ideas?

File: ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/QueryFeaturesWithArcadeExpression/QueryFeaturesWithArcadeExpression.cpp [hunk 2]
```diff
@@ -126,14 +126,16 @@ void QueryFeaturesWithArcadeExpression::showEvaluatedArcadeInCallout(Feature* fe
   profileVariables["$feature"] = QVariant::fromValue(feature);
   profileVariables["$map"] = QVariant::fromValue(m_map);
 
+
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/QueryFeaturesWithArcadeExpression/QueryFeaturesWithArcadeExpression.cpp:129]:
```suggestion
```

File: ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ReadSymbolsFromMobileStyle/ReadSymbolsFromMobileStyle.cpp [hunk 3]
```diff
@@ -241,28 +167,85 @@ void ReadSymbolsFromMobileStyle::updateSymbol(int hatIndex, int mouthIndex, int
...
 SymbolStyleSearchResultListModel* ReadSymbolsFromMobileStyle::mouthResults() const
 {
-  return m_models[1];
+  return m_models[mouthIndex];
 }
 
 SymbolStyleSearchResultListModel* ReadSymbolsFromMobileStyle::eyeResults() const
 {
-  return m_models[2];
+  return m_models[eyeIndex];
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ReadSymbolsFromMobileStyle/ReadSymbolsFromMobileStyle.cpp:233]: Good cleanup 👍. The code is more readable this way.

**Review by JamesMBallard — APPROVED** [review]
One minor thing but good overall. Please make the requested change before merging.

**Pull request merged**